project-base/app/deploy/deploy-project.sh

```diff
@@ -6,6 +6,7 @@ BASIC_AUTH_PATH="${BASE_PATH}/deploy/basicHttpAuth"
 DEPLOY_TARGET_PATH="${BASE_PATH}/var/deployment/deploy"
 CI_ENVIRONMENT_SLUG=${CI_ENVIRONMENT_SLUG}
 SENTRY_DSN=${SENTRY_DSN}
+S3_REGION=${S3_REGION:-'us-east-1'}
```

Comment:
As most of our customers are from Europe I feel like `eu-central-1` is better region

Reply:
That would require us to manually set the region for each deployed app. As we're using minIO, where the default region is set to us-east-1

![Snímek obrazovky 2023-09-05 v 11 04 51](https://github.com/shopsys/shopsys/assets/1177414/89d40c87-1088-40b8-ac1d-f6fb8213e496)